pkg/ucp/secret/inmemory/client_test.go

```diff
@@ -0,0 +1,157 @@
+/*
+Copyright 2023 The Radius Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package inmemory
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/radius-project/radius/pkg/ucp/secret"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	SecretName = "test-secret-name"
```

Comment:
Is this exported?

Reply:
I'll fix this, and in the other packages.